bin/MessagesBundle_cs.properties

```diff
@@ -218,6 +218,21 @@ lib_in_dir=Knihovny v adresáři
 Origin=Počátek
 Choose_browser = Výběr prohlížeče
 
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on it.

Reply:
ok